src/main/kotlin/org/rust/ide/docs/RustDocumentationProvider.kt

```diff
@@ -16,8 +16,14 @@ class RustDocumentationProvider : AbstractDocumentationProvider() {
 
         val name = if (element is RustMod) element.modName else element.name
         val header = if (name != null) "<pre>$name</pre>\n" else ""
+        val signature = getSignature(element)
```

Comment:
I think this can be condensed to something like `(element as? RustFnElement)?.formatSignature() ?: ""`